### PR TITLE
My updated GitHub Profile

### PR DESCRIPTION
**Pull request by alison018**
This is the process of how I created my first Pull Request

### CODE REVIEW

File: github-profiles/alina18-4913.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+*2013 -2015*
+EFL Teacher
+
+**School Kids of the future**
+*2016 -2017*
+Research Assistant
+
+# Commands that I have learnt
+
+| Commands I have Learnt | Description |
```
Comment (fran-rs) [inline github-profiles/alina18-4913.md:51]: Hay que alinear las tablas

File: github-profiles/alina18-4913.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+| ---------------------- | ----------- |
+| ls|list the content of the folder|
+| pwd|prints working directory|
```
Comment (fran-rs) [inline github-profiles/alina18-4913.md:115]: Aquí también hay que alinear tablas

Comment (alison018) [reply github-profiles/alina18-4913.md]: Muchas gracias por el feedback, ya estan alineadas

File: github-profiles/alina18-4913.md (hunk 1 above)
Comment (alison018) [reply github-profiles/alina18-4913.md]: perfecto, correccion implementada. Mil gracias

File: github-profiles/alina18-4913.md
Comment (julietalazzaro) [inline github-profiles/alina18-4913.md:1]: This file had to be removed, it has to be only in your profile folder (alina18-4913) as README.md

File: github-profiles/alina18-4913/alina18-4913.html
Comment (julietalazzaro) [inline github-profiles/alina18-4913/alina18-4913.html:1]: html file has to be called index.html, as it is the default name, and the file browser search for when looks for a site in a server.

Comment (julietalazzaro) [inline github-profiles/alina18-4913/alina18-4913.html:1]: You should add proper indentation, it makes code more easy to read. If using Visual Studio Code you can install an extension to indent automatically, like Prettier - Code formatter

**Pull request closed without merging**